src/collection/class.js

```diff
@@ -3,10 +3,7 @@ let is = require('../is');
 
 let elesfn = ({
   classes: function( classes ){
-    if( is.array( classes ) ){
-      // use a copy of the array
-      classes = classes.slice(0);
-    } else {
+    if( !is.array( classes ) ){
```

Comment:
It's probably more sensible to check if it's a string instead of if it's not an array, but this way will preserve the current error behavior if someone puts something that's not a string in here (ie throwing errors about doing string stuff to something that's not a string.)